### DISCUSSION TITLE
算是体会到了，怪不得有那么多优秀的开源项目，开发者最后会选择弃坑。

### DISCUSSION THREAD

**Discussion by viarotel** [viarotel-org/escrcpy#86] [General]
因为你的用户会纷纷化作产品经理，而且还没有工资。😂

**Comment by AkiChase** [comment]
看了眼issue，头大

**Comment by aoy2024** [comment]
别人我不清楚 我个人而言提需求说意见是为了更好用 为了让自己用的更舒服 同时也能让项目越来越完善

**Reply by viarotel** [reply]
你没有走到那一步是没有办法共情的，有时候用户提出了一个他认为很简单很容易修改的需求，但是从开发角度来说需要花费很大的精力，或者压根无法实现，这时候你如果不回复用户会觉得你不重视他的问题，这么简单的问题都不处理，如果我回复呢？我当然可以回复，但是如果这是一个热门的项目，你的用户有成百上千个，你需要拿你个人的时间来一个个回复解释，这种努力无法带给你实质性的汇报，并且有时候即使回复了也得不到用户理解这种心酸只有自己知道。😔

**Reply by aoy2024** [reply]
汲取有用的 摒弃不合适的也是有必要的 另外建议开发者做一些会员功能添加高级功能(或者推出pro版本)来给自己增加的收入 这样才能更好的可持续发展

**Reply by aoy2024** [reply]
或者可以考虑给那些小部分人需要的小众功能出收费定制版

**Reply by aoy2024** [reply]
或者某些功能可以先在pro版优先体验 大众版后续下放 我不懂哈就是个人见解

**Reply by aoy2024** [reply]
我觉得找个收入点无可厚非 这样才能可持续发展 项目有收入您能更有动力 作品也能更优秀

**Reply by viarotel** [reply]
我会考虑你的建议
